client_test.go

```diff
@@ -48,34 +47,31 @@ func TestNewClient(t *testing.T) {
 	//	assert.Equal(t, defaultBaseURL, c.BaseURL, )
 }
 
-func Test_client_CreateToken(t *testing.T) {
-	c := initClient(t)
-
-	var expire = time.Now().Add(time.Hour)
-	tt := []struct {
-		name   string
+func TestClient_CreateToken(t *testing.T) {
+	type args struct {
+		userId string
```

Comment:
it's consistent throughout the project but it would be a lint error 

`userID` would be better but should be done separately everywhere if intended.